solr-synonyms-api/synonyms/constants/__init__.py

```diff
@@ -98,6 +98,7 @@ class BCProtectedNameEntityTypes(AbstractEnum):
     LIMITED_LIABILITY_PARTNERSHIP = EntityTypes.LIMITED_LIABILITY_PARTNERSHIP.value
     COOPERATIVE = EntityTypes.COOPERATIVE.value
     COMMUNITY_CONTRIBUTION_COMPANY = EntityTypes.COMMUNITY_CONTRIBUTION_COMPANY.value
+    SOCIETY = EntityTypes.SOCIETY.value
```

Comment:
same here. Why duplicate them.  Just import them where you need them from namex-api if they are the same.  Probably the constants in syonyms shoudl be reviewed and all duplication removed.  I will create a ticket for that.